caluma/tests/snapshots/snap_test_schema.py

```diff
@@ -847,6 +846,7 @@
   source: Document
   historyDate: DateTime!
   historyType: String
+  answers(before: String, after: String, first: Int, last: Int, metaValue: [JSONValueFilterType], orderBy: [AnswerOrdering], filter: [AnswerFilterSetType], order: [AnswerOrderSetType], question: ID, search: String, createdByUser: String, createdByGroup: String, createdBefore: DateTime, createdAfter: DateTime, metaHasKey: String, questions: [ID], visibleInContext: Boolean): AnswerConnection
```

Comment:
I think this could be problematic - It will show the "current" answers, not the historical ones. Thus I'd like to see the `answers` connection being excluded in the GQL schema. (The taxonomy change was only to allow visibility reuse, right?)

Reply:
That's exactly the reason I didn't inherit from `Document`. Tthis change should not result in any changes to the schema.

Reply:
I was fearing this. So this isn't a viable change I guess.

Reply:
Yes. And this doesn't only apply to `HistoricalDocument`, but also to `HistoricalFile`.